dali/operators/numba_function/numba_func.cu

```diff
@@ -112,9 +112,41 @@ bool NumbaFuncImpl<GPUBackend>::SetupImpl(std::vector<OutputDesc> &output_desc,
   return true;
 }
 
+
+std::vector<void*> prepare_args(const dali::TensorList<dali::GPUBackend> &in, uint64_t *ptr, uint64_t shape_ptr) {
+  void** meminfo = new void*;
+  void** parent = new void*;
+  *meminfo = NULL;
+  *parent = NULL;
+  ssize_t item_size = dali::TypeTable::GetTypeInfo(in.type()).size();
+
+  vector<void*> args;
+  args.push_back((void*)meminfo);
+  args.push_back((void*)parent);
+  args.push_back((void*)new ssize_t(item_size));
+  args.push_back((void*)ptr);
+
+  size_t nitems = 1;
+  uint64_t* shapes = (uint64_t*)shape_ptr;
+  for (int i = 0; i < in.shape().ndim; i++) {
+    args.push_back((void*)(new ssize_t(shapes[i])));
+    nitems *= shapes[i];
+  }
+  
+  args.insert(args.begin()+2, (void*)new ssize_t(nitems));
```

Comment:
Maybe you can push_back the initial value of `nitems` and in L130 and update it later?

Reply:
Done